python/GafferVDBTest/SceneInterfaceTest.py

```diff
@@ -0,0 +1,48 @@
+import GafferTest
+
+import GafferVDB
+import IECore
+import GafferVDBTest
+import os
+
+class SceneInterfaceTest( GafferVDBTest.VDBTestCase ) :
+	def setUp( self ) :
+		GafferVDBTest.VDBTestCase.setUp( self )
+		self.sourcePath = os.path.join(self.dataDir, "utahteapot.vdb")
```

Comment:
Does the Utah teapot provide anything not provided by the smoke and sphere files? The other two are fairly modest in size, but the teapot is nearly 10x the size. It'd be good to avoid bloating the repo more than necessary.